### PR TITLE
Stop click events in the group name <input> from switching groups

### LINKED ISSUE #25: Clicking into "Rename Tab Group" text field causes switch to tag group
When I click into the text field for renaming a tab group and I'm not currently in this tab group, the browser switches to the tab group. This has odd side effects like not being able to type in the field or the field not being focused for typing sometimes.

### PR DESCRIPTION
**Pull request by mnoorenberghe**
Fixes #25

### CODE REVIEW

File: src/data/components/group.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ const Group = React.createClass({
           onChange: (event) => {
             this.setState({newTitle: event.target.value});
           },
+          onClick: event => event.stopPropagation(),
```
Comment (skyschub) [inline src/data/components/group.js]: Please add parenthesis and braces.

Comment (mnoorenberghe) [inline src/data/components/group.js]: You should add that to the linter… :)

Comment (skyschub) [inline src/data/components/group.js]: True! Feel free to add `arrow-parens` as an error in this PR or I'll add it afterwards. :)

**Comment by mnoorenberghe** [comment]
Braces and parenthesis added

**Comment by skyschub** [comment]
Thanks! I'll extend the eslint-configs myself.

**Comment by mnoorenberghe** [comment]
> Thanks! I'll extend the eslint-configs myself.

:+1:

**Comment by skyschub** [comment]
Merged, thanks.

**Pull request merged**